modules/core/src/main/java/org/apache/ignite/internal/processors/job/GridJobProcessor.java

```diff
@@ -1492,6 +1503,23 @@ private boolean isDeadNode(UUID uid) {
         return ctx.discovery().node(uid) == null || !ctx.discovery().pingNodeNoError(uid);
     }
 
+    /**
+     * <a href="https://issues.apache.org/jira/browse/IGNITE-6380">IGNITE-6380</a>
```

Comment:
Comments with task number are not allowed. Please, write meaningful comment if needed